common/djangoapps/custom_settings/models.py

```diff
@@ -1,12 +1,13 @@
 from django.db import models
-from django.db.models.fields import BooleanField
+from django.db.models.fields import BooleanField, CharField
```

Comment:
I think we should remove this import since we have add every field we add to model here. instead we should user `models.CharField` where we are defining fields.